kimchi/src/circuits/polynomials/keccak/witness.rs

```diff
@@ -355,16 +355,25 @@ impl Chi {
 /// Values involved in Iota permutation step
 pub struct Iota {
     state_g: Vec<u64>,
+    rc: Vec<u64>,
 }
 
 impl Iota {
-    fn create(state_f: Vec<u64>, round: usize) -> Self {
+    pub fn create(state_f: &[u64], round: usize) -> Self {
         let rc = Keccak::sparse(RC[round]);
```

Comment:
And following [this](https://github.com/o1-labs/proof-systems/pull/1558/files#r1430165407), we could get rid of this line, and have it static somewhere.